### PR TITLE
Fix: #1407: This adds Release notes for 1.11.3 

### LINKED ISSUE #1407: Release notes for 1.11.3
### Release Notes for 1.11.3

- Added support for U2F/WebAutn hardware security keys. (https://github.com/brave/brave-ios/issues/1406)
- Fixed app crash when switching from Private mode to normal mode. (https://github.com/brave/brave-ios/issues/1309)

### PR DESCRIPTION
**Pull request by srirambv**
## Submitter Checklist:

- [X] Submitted a [ticket](https://github.com/brave/brave-ios/issues) for my issue if one did not already exist.
- [x] My patch or PR title has a standard commit message that looks like `Fix #123: This fixes the shattered coffee cup!` (or `No Bug: <message>` if no relevant ticket)
- [x] *Unit Tests* are updated to cover new or changed functionality
- [x] User-facing strings use `NSLocalizableString()`
- [x] New files have MPL-2.0 license header.

## Test Plan:
Fixes: #1407

## Reviewer Checklist:

- [ ] PR is linked to an issue via [Zenhub](https://www.zenhub.com/extension).
- [ ] Issues are assigned to at least one epic.
- [X] Issues include necessary QA labels:
  - [x] `QA/(Yes|No)`
  - [x] `release-notes/(include|exclude)`
  - [ ] `bug` / `enhancement`
- [ ] Necessary security reviews have taken place.
- [ ] Adequate test coverage exists to prevent regressions.
- [ ] Adequate test plan exists for QA to validate (if applicable)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## [1.11.3](https://github.com/brave/brave-ios/releases/tag/v1.11.3)
+
+ - Added support for U2F/WebAutn hardware security keys. ([#1406](https://github.com/brave/brave-ios/issues/1406))
```
Comment (kjozwiak) [inline CHANGELOG.md]: nit: `WebAutn` should be `WebAuthn`

maybe we can word it as:

```
Added support for U2F/WebAuthn using hardware security keys.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
...
+ - Fixed app crash when switching from Private mode to normal mode. ([#1309](https://github.com/brave/brave-ios/issues/1309))
```
Comment (kjozwiak) [inline CHANGELOG.md:6]: nit: I don't think we need to have `app` in there. Maybe reword to something like:

```
Fixed crash when switching from private browsing to regular browsing mode.
```

**Review by kjozwiak — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (srirambv) [reply CHANGELOG.md:6]: Helps differentiate between app crash vs webview crash. The same convention is used on Android as well.

Comment (srirambv) [reply CHANGELOG.md:6]: Normal tabs/mode is the generally used convention on all platforms and specifically on iOS. Should probably use the same here as well.

**Review by kjozwiak — APPROVED** [review]
LGTM 👍
